yocto/shortlog.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# vagrant-cookbook
+# Copyright (C) 2017 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+#
+#
+# Usage: shortlog.sh <yocto_build_dir>
+#
+# Source the environment and create git shortlog for each layer .
+#
+
+YOCTO_BUILD_DIR="$1"
+SHORT_LOG=/vagrant/reports/shortlog.txt
+
+get_changes() {
+    folder="$1"
+    layer=${folder#$YOCTO_BUILD_DIR../}
+    cd /vagrant
```

Comment:
Here again, I wouldn't presume that the git repo is in a directory called `/vagrant`